metadata-io/src/main/java/com/linkedin/metadata/timeline/TimelineServiceImpl.java

```diff
@@ -166,10 +171,10 @@ public List<ChangeTransaction> getTimeline(@Nonnull final Urn urn, @Nonnull fina
         .filter(aspectSpec -> !aspectSpec.isTimeseries())
         .map(AspectSpec::getName)
         .collect(Collectors.toSet());
-    List<EbeanAspectV2> aspectsInRange = this._entityDao.getAspectsInRange(urn, fullAspectNames, startTimeMillis, endTimeMillis);
+    List<EntityAspect> aspectsInRange = this._aspectDao.getAspectsInRange(urn, fullAspectNames, startTimeMillis, endTimeMillis);
```

Comment:
I think we should be able to push these DAO specific methods into an EntityService layer wrapper, what do you think? This would reduce the need for the split tests. Was this avoided due to level of effort or just trying to limit scope creep for changes?

Reply:
I _would_ like to kill the need for TimelineService to have split tests. My mind was never on it because the diff of introducing EntityAspect was already spiralling out of control. 

Also, public methods in EntityService tend to wrap everything in RecordTemplates and if TimelineService used EntityService instead of the DAO - I assumed it (and the differs) would have to be rewired to use them instead of EntityAspect. I didn't feel like finding out how many changes that would be 😄